Resource definitions.json not found

Hi, I run into an issue when running a project with embedded tomcat in jar with Java 11. 

It causes the following Exception:

`
Caused by: java.lang.IllegalArgumentException: resource definitions.json not found.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:217)
	at com.google.common.io.Resources.getResource(Resources.java:195)
	at org.xrpl.xrpl4j.codec.binary.definitions.DefaultDefinitionsProvider.lambda$new$0(DefaultDefinitionsProvider.java:26)
`

And is coming from DefaultDefinitionProvider.java:26. The definitions.json file is in the jar, just the classLoader won't resolve it.

--- a/xrpl4j-binary-codec/src/main/java/org/xrpl/xrpl4j/codec/binary/definitions/DefaultDefinitionsProvider.java
+++ b/xrpl4j-binary-codec/src/main/java/org/xrpl/xrpl4j/codec/binary/definitions/DefaultDefinitionsProvider.java
@@ -23,7 +23,7 @@ public class DefaultDefinitionsProvider implements DefinitionsProvider {
 
     this.supplier = Suppliers.memoize(() -> {
       try {
-        return objectMapper.readerFor(Definitions.class).readValue(Resources.getResource("definitions.json"));
+        return objectMapper.readerFor(Definitions.class).readValue(Resources.getResource(DefaultDefinitionsProvider.class, "/definitions.json"));
       } catch (IOException e) {
         throw new IllegalStateException("Cannot read definition.json file", e);
       }

